src/snow_rs/modis/variable_from_modis.py

```diff
@@ -37,42 +37,67 @@ def argument_parser():
              'folder with the water year. Example: 2018'
              '\n'
     )
+    
+    parser.add_argument(
+        '--calendar_year_src_dir',
```

Comment:
How about using `--calendar-year` as the flag to trigger that instead of a water year?

Personally, my head usually hurts having to think of the negated case.

Reply:
Yes--that is weird. If I understand correctly, we should make `--water-year` and `--calendar-year` optional, where the argument for each is the year, (no boolean) and the user supplies whichever is needed?

Reply:
We could use something like [argument choices](https://docs.python.org/3/library/argparse.html#choices) and force the user that way to pick one?

Reply:
Alternatively we could also get rid of the date loop and assume that the user wants to convert all the files in the source directory. Would put a little more burden on the data preparation side, but simplify the [conversion logic](https://github.com/UofU-Cryosphere/snow-rs/blob/cb835cebd5245c2ec2de3f00e95e2ee5edc70056/src/snow_rs/modis/matlab_to_geotiff.py#L37)

Reply:
Supplying a calendar year to `--water-year` is also odd. I was also avoiding any changes to `lib/command_line_helpers.py`, given it is reused elsewhere? I could also just move `add_water_year_option` arg to `lib/modis_geotiff.py` and change the name to `--year`

Reply:
You could remove that helper all together. I extracted this logic from a different library, where it was reused. Here it's fair game to chop this up!

Then we could do a choice argument that accepts `--year-format [calendar|water]`

Reply:
Ok, I've added a `--year` and `--year-format` arg. Works fine for me with the new changes.